src/explore-education-statistics-admin/src/pages/release/content/components/KeyStatistics.tsx

```diff
@@ -29,20 +101,23 @@ const KeyStatistics = ({ release, isEditing }: KeyStatisticsProps) => {
             Any data blocks with more than one value cannot be selected as a key
             statistic.
           </WarningMessage>
-          <AddKeyStatistics release={release} />
+          <ButtonGroup className={styles.buttons}>
+            <AddKeyStatistics release={release} />
+            {keyStatisticsBlocks.length > 1 && <ReorderKeyStatisticsButton />}
+          </ButtonGroup>
         </>
       )}
-      <KeyStatContainer>
-        {release.keyStatisticsSection.content
-          .filter(block => block.type === 'DataBlock')
-          .map(block => (
+      {!isReordering && (
```

Comment:
Is it possible to combine the reordering and non-reordering states into a single declaration? 

Ideally it'd be good to have something like `EditableAccordion` where we only declare things once and just toggle the reordering state via props. Too much duplication like this can easily lead to bugs in the future.

Reply:
Good call. I've combined them, it was a little bit fiddly because of the flexbox so I've moved the styles to a common file to avoid having to duplicate them and not break the other places where the KeyStatColumn is used.